plugins/compiled-markdown-directive.js

```diff
@@ -3,7 +3,9 @@ import xss from 'xss'
 import marked from 'marked'
 
 function compileMarkdown(target, markdown) {
-  target.innerHTML = xss(marked(markdown))
+  target.innerHTML = xss(marked(markdown), {
+    whiteList: Object.assign(xss.whiteList, { summary: [] }),
+  })
```

Comment:
I'd do it like this:
```js

const options = {
    whiteList: {
        summary: [],
    },
};
const configuredXss = new xss.FilterXSS(options);

function compileMarkdown(target, markdown) {
  target.innerHTML = configuredXss.process((marked(markdown))
}
```

Reply:
Seems good, however, it only missed something to also have the original whitelist's values. I figured out a better way to do that without Object.assign